vrc-get-gui/locales/zh_hans.json5

```diff
@@ -280,7 +280,7 @@
     "projects:toast:unity migrated": "项目已迁移至 Unity 2022",
 
     "projects:manage:dialog:confirm changes description": "您正在对项目进行以下更改。",
-    "projects:manage:dialog:note breaking changes": "您更改的软件包可能会导致一些破坏性的更改。<br/>项目中的资产、配置和工具可能不兼容。<br/>请查阅兼容性信息，谨慎升级。",
+    "projects:manage:dialog:note breaking changes": "“部分软件包存在破坏性变更。<br/>项目中的资产、配置和工具可能不兼容。<br/>请查阅兼容性信息，谨慎升级。",
```

Comment:
It looks there is no closing `“` or added it by mistake. Please let me know if this is OK in Chinese.